src/utils/parsingModel.json

```diff
@@ -2696,5 +2696,201 @@
         }
       ]
     }
+  },
+ "Activité du journal des accès/Activités _ liste des services Google auxquels vos.csv": {
```

Comment:
Il me parait bizarre le nom du path, c'est comme s'il était incomplet ? C'est le bon nom ? 

Sinon en task, il faudra dupliquer les objects en format EN. Par exemple, pour ce fichier j'ai : `Access log activity/Activities – A list of Google services accessed by.csv`

Reply:
C'est le nom que j'ai à l'export je n'ai pas modifié.

Je rajoute EN

Reply:
Tu pourras tester si ça parse bien avec ton fichier ?